sdk/src/main/java/com/ingresse/sdk/model/response/UpdateTransferJSON.kt

```diff
@@ -0,0 +1,22 @@
+package com.ingresse.sdk.model.response
+
+data class UpdateTransferJSON(
+    val saleTicketId: Int? = 0,
+    val user: User? = null,
+    val status: String? = "",
+    val id: String? = ""
+)
+
+data class User(
```

Comment:
I think is always good put JSON in the end of class name of response files. It's better to recognize when the object it's from api or project. Apply in the others classes too.